sbpy/activity/gas/tests/test_prodrate.py

```diff
@@ -1,3 +1,15 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+import pytest
```

Comment:
pytest import is needed if you are using the context manager or decorator but perhaps not needed here